app/helpers/application_helper.rb

```diff
@@ -1,21 +1,28 @@
 module ApplicationHelper # :nodoc:
-  def markup(text, filters = [])
+  def markup(text, options={})
     return unless text.present?
 
-    pipeline_filters =
-      if filters.empty?
-        [
-          HTML::Pipeline::Dradis::FieldableFilter,
-          HTML::Pipeline::Dradis::TextileFilter,
-          HTML::Pipeline::SanitizationFilter,
-          HTML::Pipeline::AutolinkFilter,
-          HTML::Pipeline::Dradis::CodeHighlightFilter
-        ]
-      else
-        filters
+    context = {}
+    pipeline_filters = [
+      HTML::Pipeline::Dradis::FieldableFilter,
+      HTML::Pipeline::Dradis::TextileFilter,
+      HTML::Pipeline::SanitizationFilter,
+      HTML::Pipeline::AutolinkFilter,
+      HTML::Pipeline::Dradis::CodeHighlightFilter
+    ]
+
+    if options[:liquid]
+      context[:liquid_assigns] = liquid_assigns
+      pipeline_filters.insert(1, HTML::Pipeline::Dradis::LiquidFilter)
+    end
+
+    if options[:filters] && options[:filters].any?
```

Comment:
Can be simplified to

```suggestion
    if options[:filters].present?
```